environments/prod/prod.tfvars

```diff
@@ -916,5 +916,13 @@ frontends = [
         selector       = "token"
       },
     ]
-  }
+  },
+  {
+    name             = "pcq"
+    mode             = "Detection"
+    custom_domain    = "equality-and-diversity.platform.hmcts.net"
+    backend_domain   = ["firewall-prod-int-palo-prod.uksouth.cloudapp.azure.com"]
+    certificate_name = "equality-and-diversity-platform-hmcts-net"
+    www_redirect     = true
```

Comment:
Is this intentional ?

Reply:
I am guessing it's not as i cannot see dns entry for it.

Reply:
No, we don't. www_redirect has been removed.